GitPrompt.ps1

```diff
@@ -306,8 +306,4 @@ $PoshGitVcsPrompt = {
     Write-GitStatus $GitStatus
 }
 
-# Install handler for removal/unload of the module
 $Global:VcsPromptStatuses += $PoshGitVcsPrompt

```

Comment:
If we do that, should we also move the decl/assignment of `$PoshGitVcsPrompt` just above this line as well?